src/main/java/com/jvms/i18neditor/editor/menu/NextTranslationErrorMenuItem.java

```diff
@@ -0,0 +1,25 @@
+package com.jvms.i18neditor.editor.menu;
+
+import java.awt.Toolkit;
+import java.awt.event.KeyEvent;
+
+import javax.swing.JMenuItem;
+import javax.swing.KeyStroke;
+
+import com.jvms.i18neditor.editor.TranslationTree;
+import com.jvms.i18neditor.util.MessageBundle;
+
+/**
+ * This class represents a menu item to navigate to the next translation error in the translation tree.
+ * 
+ * @author Fr�d�ric Courchesne
+ */
+public class NextTranslationErrorMenuItem extends JMenuItem {
+	//private final static long serialVersionUID = 7316102121075733726L;
```

Comment:
_NextTranslationErrorMenuItem_ has been copied/pasted from _ExpandTranslationsMenuItem_ and will require a unique ID.